configuration/customize-emails.md

```diff
@@ -4,14 +4,18 @@ description: This article explains how to customize e-mails within SysKit Point.
 
 # Customize E-mails
 
-SysKit Point **allows you to customize e-mails** that will be **sent to site owners** as a part of **Access Review and Lifecycle Management Automation**.
+SysKit Point **provides the option to customize e-mails** that are **sent to site owners** as a part of **Access Review and Lifecycle Management Automation**. 
 
-You can customize the following aspects of e-mails:
+These emails can be **scheduled** to send at a specific time period. Certain areas can be modified to suit your needs. 
```

Comment:
ne bih tu baš koristio riječ scheduled. 
za Access Review se može definirati dal će se odrađivati svaki mjesec, dva, itd., ali ne možeš definirati konkretni datum za pojedini Access Review, jer je složeno da se ne šalju vikendom itd., pa nikad baš neće biti točno mjesec dana između njih
Lifecycle management mail nema ni taj dio sa redovitim ponavljanjem nego čim neki workspace 'postane' inactive prema postavkama u settingsima, on će se poslati

Reply:
Ustvari bih samo maknuo prvu rečenicu za sad. These emails can be **scheduled** to send at a specific time period
Kad složimo article sa popisom i opisom svih mailova, možemo odavde linkati na taj detaljniji članak gdje će se vidjeti kada se koji mail šalje i zašto